### ISSUE TITLE
BMLT tabs not displaying properly

### ISSUE THREAD

**Issue by chrisr37** [bmlt-enabled/crouton#409]
**Describe the bug**
When going to https://nnerna.org/meeting-list-2/ the BMLT tabs are not displaying properly
**To Reproduce**
1. Go to https://nnerna.org/meeting-list-2/ 
2. Scroll down to the tabs just below the map

**Expected behavior**
Expect to see the tabs displating properly

**Screenshots**
![image](https://user-images.githubusercontent.com/124472967/216794124-9264c0a8-576c-4915-a2db-201e3d08528b.png)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser Chrome 
 - Version 109.0.5414.120 (Official Build) (64-bit)

**Additional context**
Think this has happened a few years ago, not sure what the fix was

**Comment by tempsaint** [comment]
Did you try changing Wordpress themes by any chance?

**Comment by chrisr37** [comment]
No. Still using Mesmerize. I did download a theme recently, but did not activate it, In the WP Site Health page, I had been getting an error message to have a default theme, but only had one theme, so did not understand why I was getting the error message.
I watched a video on how to get rid of the error message and they recommended downloading a theme and then the one you are using would be the default and the error meeage would go away and that is what happened. I never activated the new theme. Could that be related?

**Comment by otrok7** [comment]
It seems like this has been resolved.  Is that correct?

**Comment by otrok7** [comment]
From the original poster:

Yes, 6 months ago, someone looked at it and added some code and it fixed it. This is the message from March: Hello, someone looked at the CSS for the [nnerna.org](http://nnerna.org/) meeting list header display issue and fixed it. Very fortunate. This is what he added. I found the magic CSS to correct it and added it to the Additional CSS in the Customizer.
This is what I added:
div:after, div:before {
  display: none;
}
